<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Poles</class>
 <widget class="QWidget" name="Poles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="poleValueLabel">
       <property name="text">
        <string>Pole</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="changeValueButton">
       <property name="text">
        <string>Change Value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deletePoleButton">
       <property name="text">
        <string>Delete Pole</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
